frontend/src/pages/PassingTest/ConfirmLeavePage/ConfirmLeavePage.tsx

```diff
@@ -16,7 +17,6 @@ interface IProps {
 }
 
 const CONFIRM_MESSAGE = 'Are you sure you want to leave this page?';
```

Comment:
is it possible to move this to the constants file?

Reply:
sure